package.json

```diff
@@ -8,67 +8,68 @@
     "ui toolkit",
     "v1"
   ],
-  "main": "dist/scripts/helix-ui.js",
-  "module": "dist/scripts/helix-ui.es.js",
+  "main": "dist/js/helix-ui.cjs.js",
+  "module": "dist/js/helix-ui.esm.js",
+  "unpkg": "dist/js/helix-ui.js",
   "files": [
     "dist/"
   ],
   "devDependencies": {
-    "@webcomponents/webcomponentsjs": "^2.0.4",
-    "babel-core": "^6.26.0",
-    "babel-loader": "^7.1.2",
-    "babel-plugin-external-helpers": "^6.22.0",
-    "babel-preset-env": "^1.6.1",
+    "@babel/core": "7.x",
+    "@babel/plugin-external-helpers": "7.x",
+    "@babel/preset-env": "7.x",
+    "@webcomponents/webcomponentsjs": "2.x",
+    "babel-loader": "8.x",
     "browser-sync": "2.x",
     "chai": "^4.2.0",
-    "cheerio": "^1.0.0-rc.2",
+    "cheerio": "^1.0.0-rc.3",
     "clean-css": "^4.1.9",
-    "eslint": "4.x",
-    "eslint-loader": "2.x",
+    "eslint": "6.x",
+    "eslint-loader": "3.x",
     "esm": "^3.0.84",
     "express": "^4.16.3",
-    "front-matter": "^2.2.0",
-    "fs-extra": "6.x",
+    "front-matter": "3.x",
+    "fs-extra": "8.x",
     "gh-pages": "^2.0.0",
-    "globby": "8.x",
+    "globby": "10.x",
+    "gulp": "^4.0.2",
     "highlight.js": "^9.12.0",
     "jsdoc": "^3.5.5",
-    "jsdom": "^12.2.0",
-    "json5": "1.x",
+    "jsdom": "15.x",
+    "json5": "2.x",
     "less": "3.x",
-    "less-loader": "^4.0.5",
+    "less-loader": "5.x",
     "lodash": "^4.17.15",
-    "mocha": "^5.2.0",
+    "mocha": "6.x",
     "moment": "^2.18.1",
     "node-sass": "^4.12.0",
     "nunjucks": "^3.0.1",
-    "nyc": "^13.0.1",
-    "raw-loader": "^0.5.1",
-    "rollup": "0.60",
-    "rollup-plugin-babel": "^3.0.2",
-    "rollup-plugin-commonjs": "9.x",
-    "rollup-plugin-eslint": "^4.0.0",
+    "nyc": "14.x",
+    "raw-loader": "3.x",
+    "rollup": "1.x",
+    "rollup-plugin-babel": "4.x",
+    "rollup-plugin-commonjs": "10.x",
+    "rollup-plugin-eslint": "7.x",
     "rollup-plugin-html": "^0.2.1",
-    "rollup-plugin-json": "3.x",
-    "rollup-plugin-node-resolve": "^3.0.0",
-    "rollup-plugin-uglify": "4.x",
-    "rollup-pluginutils": "^2.0.1",
-    "sinon": "^6.3.5",
+    "rollup-plugin-json": "4.x",
+    "rollup-plugin-node-resolve": "5.x",
+    "rollup-plugin-terser": "^5.1.2",
+    "rollup-pluginutils": "2.x",
+    "sinon": "7.x",
     "sinon-chai": "^3.2.0",
     "stylelint": "^11.0.0",
     "svg-inline-loader": "^0.8.0",
-    "tar": "^4.4.4",
-    "uglify-es": "^3.2.2",
+    "tar": "5.x",
     "webpack": "4.x",
-    "whatwg-fetch": "^2.0.4"
+    "whatwg-fetch": "3.x",
+    "yargs": "^14.2.0"
   },
   "bundledDependencies": [
     "@webcomponents/webcomponentsjs"
   ],
   "scripts": {
     "apidocs": "jsdoc -c jsdoc.conf.json",
-    "clean": "rm -fr node_modules",
```

Comment:
original behavior of `clean` provided no useful utility (you can run `rm -fr node_modules` just as easily as `yarn clean`)